qiita_db/support_files/patches/26.sql

```diff
@@ -0,0 +1,15 @@
+-- Jun 11, 2015
+
+-- Updating FASTA-Sanger -> FASTA_Sanger, needed so we can put restrictions on
+-- what kind of files the user can select in the GUI
+
+UPDATE qiita.filetype SET type='FASTA_Sanger' WHERE type = 'FASTA-Sanger';
+
+-- Adding new filetype and new illumina processing params if they do not exists
+-- taken from: http://stackoverflow.com/a/13902402
```

Comment:
Minor, but can you move this comment to the line prior to the insert in line 13?
And `taken` -> `adapted`